File: Downloads/GITHUB/Exit project/Full stack UI/FullStack.UI/src/app/Models/Order.ts
export type Decimal = number;

export interface Order {
    orderID: string;
    userID: string;
    productID: string;
    totalMoneyPaid: number | Decimal;
    orderTime: Date;
    orderDate: Date;
    productQuantity:number;
    // Include other necessary properties
  }
  

File: Downloads/GITHUB/Exit project/Full stack UI/API/DataAccessLayer/models/Order.cs
using DataAccessLayer.models;
using System;
using System.Collections.Generic;

namespace DataAccessLayer.Models
{
	public class Order
	{
		/// <summary>
		public Guid OrderID { get; set; }
		/// </summary>
		public string UserID { get; set; }
		public string productID { get; set; }

		public int productQuantity { get; set; }

		public long TotalMoneyPaid { get; set; }
		public DateTime OrderTime { get; set; }
		public DateTime OrderDate { get; set; }
		// Include other necessary properties
	}
}
